kernel/libs/aster_page_allocator/src/buddy_allocator.rs

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: MPL-2.0 OR MIT
+// To be specific, the original source code is from [buddy_system_allocator](https://github.com/rcore-os/buddy_system_allocator),
```

Comment:
Insert a blank line between 1 and 2. Also break the line before the link to improve readability.